helper.sh

```diff
@@ -234,6 +269,11 @@ if (( ${command_num} > 1 )); then
     echo "Usage: helper.sh {--init|--build|--start|--refresh|--monitor|--stop|--down|--restart|--nuke|--execute|--bump|--docker} [-v]"; exit 1
 fi
 
+if [[ $HOSTNAME != "appserver" ]]; then
```

Comment:
Does this work in a non-vagrant environment?

Reply:
no, but as per the other comments yesterday I have no more energy to support non-vagrant - and after all, it's very clear in the readme that non-vagrant shouldn't be the case

Reply:
this line has to be edited further, though

Reply:
"It works fine on my machine", but I understand

Reply:
what do you mean? vagrant is exactly to avoid the "works on my machine"

Reply:
If it works for people with Docker and we don't have to give active support, it's fine what people use imo. Or at least; I'm using the no-vagrant version for my server. No need to update this though, I can always comment this out on my machine. 
This guard will help more people than it will cost me time, so it's fine this way